### ISSUE TITLE
g++ not found during package tests

### ISSUE THREAD

**Issue by aseyboldt** [conda-forge/aesara-feedstock#78]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

It seems that aesara isn't able to locate the g++ compiler that was installed as a runtime dependency.
At least it reports this in the logs: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=523596&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=1060

### Installed packages

```shell
Refers to build log on github actions
```

### Environment info

```shell
Refers to build log on github actions
```

**Comment by brandonwillard** [comment]
@maresb

**Comment by maresb** [comment]
Should be fixed by #77. Compare with [these logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=530549&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=1473).

**Comment by brandonwillard** [comment]
> Should be fixed by #77. Compare with [these logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=530549&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=1473).

OK, closing for now.
